electron/src/update/Squirrel.ts

```diff
@@ -17,11 +17,15 @@
  *
  */
 
-import {spawn} from 'child_process';
+// https://github.com/atom/atom/blob/master/src/main-process/squirrel-update.js
```

Comment:
Let's link to a specific commit rather than to the "master" branch (which might change over time).